MAINTENANCE.md

```diff
@@ -0,0 +1,57 @@
+# Maintenance
+This document outlines basic procedures for maintenance of issues, PRs, and releases.
+
+## Issues and Pull Requests
+Issues and pull requests are at the core of hapi's approach to documentation and messaging to the broader community.  They are rigorously labeled, associated with releases, and assigned to collaborators responsible for their resolution.  Official discussions, release notes, announcements, and community support all generally take the form of issues.  Finally, the source of truth for our changelogs are also managed through issues.
+
+### Labels
+Labels are standardized through [organization defaults](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-default-labels-for-repositories-in-your-organization).  Here are some notable ones:
+ - `support` - requests for clarification or help.
+ - `bug` - reports of unintended behavior.
+ - `security` - indication of security implications.
+ - `feature` - requests for new functionality.
+ - `release notes` - documentation of changes in major releases, sometimes accompanied by a migration guide.
+ - `documentation` - non-code related changes to the readme, API docs, or similar.
+ - `dependency` - changes to underlying or peer dependencies.
+ - `breaking changes` - indication of a breaking change in behavior, principally for major releases.
+ - `help wanted` - signals that it's welcome for any community member to offer a PR or otherwise help to resolve the issue.
+
+Every issue and pull request should have at least one label attributed to it.  The labels for an issue may change over time.  When a pull request addresses a particular issue, it will often have the same labels as that issue.
+
+### The Connection to Releases
+We associate PRs and issues to released versions of our modules in order to leave a paper trail for users that includes code, documentation, and discussion.  It can also be interpreted programmatically to e.g. display automatically-generated changelogs on the website as can be seen [here for boom](https://hapi.dev/module/boom/changelog/).
+
+#### Milestones
+When a PR is merged or issue is closed, it should be associated with a [milestone](https://github.com/hapijs/hapi/milestones) for the version of the module in which the corresponding work will be released.  All merged PRs should be associated with a milestone, and all issues resolved by a merged PR or a commit should have a milestone.  Milestones are named by the full semver version of the release, e.g. `2.1.0`, `0.1.1`, `9.0.0`.  When the version is published, that milestone should be closed.  There should never be zero open milestones for a given module: when one milestone is closed, a new one is created for the next patch version.  For example, once `2.1.0` is released then the milestone for that version will be closed and a new milestone for `2.1.1` will be created.
```

Comment:
It might be worth adding a sentence stating that if things aren't part of a milestone, they won't show up on the website. That was unpleasant to discover the hard way 😄

Reply:
```suggestion
When a PR is merged or issue is closed, it should be associated with a [milestone](https://github.com/hapijs/hapi/milestones) for the version of the module in which the corresponding work will be released.  All merged PRs should be associated with a milestone, and all issues resolved by a merged PR or a commit should have a milestone.  Milestones are named by the full semver version of the release, e.g. `2.1.0`, `0.1.1`, `9.0.0`.  When the version is published, that milestone should be closed.  There should never be zero open milestones for a given module: when one milestone is closed, a new one is created for the next patch version.  For example, once `2.1.0` is released then the milestone for that version will be closed and a new milestone for `2.1.1` will be created.

This organization of issues into milestones is central to generating the module changelogs found on [hapi.dev](https://hapi.dev), and to creating transparency around what is contained in each release.
```